src/HtmlGenerator/Pass1-Generation/GenerateFromBuildLog.cs

```diff
@@ -16,34 +16,6 @@ public class GenerateFromBuildLog
         public static readonly Dictionary<string, string> AssemblyNameToFilePathMap =
             new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
 
-        public static void IndexLogFile(string logFile, string serverPath = null, string solutionRoot = null)
-        {
-            var invocations = LogAnalyzer.GetInvocations(
-                options: new LogAnalyzer.Options
-                {
-                    CheckForOrphans = false,
-                    CheckForMissingOutputBinary = false,
-                    SanityCheck = false
-                },
-                logFiles: new[]
-                {
-                    logFile
-                });
-
-            var buildlogInvocations = invocations.Select(i => new CompilerInvocation()
-            {
-                CommandLineArguments = i.CommandLine,
-                OutputAssemblyPath = i.OutputAssemblyPath,
-                ProjectFilePath = i.ProjectFilePath,
-                ServerPath = serverPath,

```

Comment:
This was the only method in which a non-blank value could have ended up in SolutionGenerator.ServerPath.

Reply:
Another option would be to remove the optional parameter and leave the rest of the method.